src/pages/workspace/WorkspaceInviteMessagePage.tsx

```diff
@@ -100,14 +86,22 @@ function WorkspaceInviteMessagePage({
         // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
     }, []);
 
+    useEffect(() => {
```

Comment:
This caused issue where the welcome message re-appears when deleted - More information here https://github.com/Expensify/App/issues/49996